shell/platform/android/io/flutter/plugin/editing/TextInputPlugin.java

```diff
@@ -252,7 +252,8 @@ private static int inputTypeFromTextInputType(
       textType |= InputType.TYPE_TEXT_FLAG_MULTI_LINE;
     } else if (type.type == TextInputChannel.TextInputType.EMAIL_ADDRESS) {
       textType |= InputType.TYPE_TEXT_VARIATION_EMAIL_ADDRESS;
-    } else if (type.type == TextInputChannel.TextInputType.URL) {
```

Comment:
Consider adding a test to make sure `TextInputType.WEB_SEARCH` remaps to `InputType.TYPE_TEXT_VARIATION_URI` on Android.

Reply:
Very good idea